h2o-genmodel/src/main/java/hex/genmodel/utils/Distribution.java

```diff
@@ -0,0 +1,60 @@
+package hex.genmodel.utils;
+
+import java.util.Map;
+
+import static hex.genmodel.utils.Distribution.Family.bernoulli;
+import static hex.genmodel.utils.Distribution.Family.modified_huber;
+import static hex.genmodel.utils.Distribution.Family.multinomial;
+import static java.lang.Math.exp;
+
+/**
+ * Copy of `hex.Distribution.Family`.
+ */
+public class Distribution {
```

Comment:
Alright check the latest commit -- now DistributionFamily is in hex.genmodel.utils, and all link functions as well.